pkg/plugins/common.go

```diff
@@ -73,3 +75,15 @@ func download(url string) (string, error) {
 	bytes, err := ioutil.ReadAll(resp.Body)
 	return string(bytes), err
 }
+
+func getHttpClient() *http.Client {
+	return &http.Client{
+		Transport: &http.Transport{
+			Proxy: http.ProxyFromEnvironment,
+			TLSClientConfig: &tls.Config{
```

Comment:
just wondering if we should have a high-level general proxy section in the cloud config syntax - this will work only if the system environment are configured correctly

Anyhow, looks good here

Reply:
I thought of that too...but wasnt sure if it belongs with yip or with tool using yip e.g. elemental which defines proxies etc in its config

Reply:
I think it somehow belongs here and elemental should respect the config coming from cloud-init, at the end this potentially ties it to other plugins that might be able to respect that